app/scripts/metamask-controller.js

```diff
@@ -2381,6 +2391,31 @@ export default class MetamaskController extends EventEmitter {
     return this.keyringController.fullUpdate();
   }
 
+  /**
+   * Submits a user's encryption key to log the user in via login token
+   */
+  async submitEncryptionKey() {
+    this.appStateController.store.updateState({
+      isAttemptingSessionLogin: true,
+    });
+
+    try {
+      const { loginToken } = await browser.storage.session.get(['loginToken']);
+      if (loginToken) {
+        await this.keyringController.submitEncryptionKey(loginTokens);
```

Comment:
I believe this should be `loginToken`

![logintokens-not-defined](https://user-images.githubusercontent.com/54408225/198875191-c1e7008f-af77-4974-a23a-d79ad1fc8d5b.png)